usaspending_api/etl/management/commands/elasticsearch_indexer.py

```diff
@@ -227,7 +227,7 @@ def set_config(passthrough_values: list, arg_parse_options: dict) -> dict:
             "field_for_es_id": "transaction_id",
             "initial_datetime": default_datetime,
             "max_query_size": settings.ES_TRANSACTIONS_MAX_RESULT_WINDOW,
-            "optional_predicate": """WHERE "update_date" >= '{starting_date}'""",
+            "optional_predicate": """WHERE "etl_update_date" >= '{starting_date}'""",
```

Comment:
Why use this one here? 
In the `es_rapidloader.py`, it sticks to just comparing to `UTM.update_date`, which is the update date of the `transaction_normalized` record. By using `etl_update_date` it may end up pulling in transactions even if there was no change to the transaction, but because the transaction's award was updated -- which seems to not be the desired outcome (and is avoided in the rapidloader).

Reply:
> By using etl_update_date it may end up pulling in transactions even if there was no change to the transaction, but because the transaction's award was updated 

Yes, that is the needed behavior

Reply:
Spoke about this briefly -- looks like this is the one we want to use (since we're pulling fields from the award down into the transaction-level ES index record). 

That being the case, the `es_rapidloader.py` should be updated to use `etl_update_date` to get the next batch of changes. It'd be helpful but not required too if the python param in the function was also renamed from `update_date=` to `etl_update_date=`

@tony-sappe

Reply:
already done